docs/pages/Troubleshooting.md

```diff
@@ -51,4 +51,17 @@ UI5 + Karma:
 
 
 !!! warning
-    The combination of the `UI5_LOG_LVL` environment variable with the `--log-level` CLI parameter might lead to unexpected results; they should be used interchangeably but not together. The CLI parameter takes precedence over the `UI5_LOG_LVL` environment variable.
\ No newline at end of file
+    The combination of the `UI5_LOG_LVL` environment variable with the `--log-level` CLI parameter might lead to unexpected results; they should be used interchangeably but not together. The CLI parameter takes precedence over the `UI5_LOG_LVL` environment variable.
+
+### Changing UI5 Tooling's Data directory
+
+UI5 Tooling's data directory is by default at `~/.ui5`. It's the place where the framework artefacts are stored.
+In some cases and environments this is not a convenient location and the user needs to provide a better one. The path to it is provided via the `UI5_DATA_DIR` environment variable.
+
+Example:
```

Comment:
On what system?

Reply:
I've updated the samples. Added windows style, just because the pathnames there are a bit different.
Hopefully, now should be more understandable